typed-racket-lib/typed-racket/base-env/prims-contract.rkt

```diff
@@ -391,9 +393,13 @@
            #,(if (attribute ne)
                  (internal (syntax/loc stx (define-type-alias-internal ty (Opaque pred))))
                  (syntax/loc stx (define-type-alias ty (Opaque pred))))
-           #,(ignore #'(define pred-cnt
-                         (or/c struct-predicate-procedure?/c
-                               (any-wrap-warning/c . c-> . boolean?))))
+           #,(if (attribute unsafe)
+                 (ignore #'(define pred-cnt
```

Comment:
Shouldn't it still attempt to check that it is a predicate? I was trying to make the change in behavior as minimal as possible for this